Project.toml

```diff
@@ -0,0 +1,14 @@
+name = "Nullables"
+uuid = "4d1e1d77-625e-5b40-9113-a560ec7a8ecd"
+version = "1.0.0"
```

Comment:
Bumping version from 0.0.8 to 1.0.0 means that some packages will have to change they version requirements. Is that really worth it? Do you plan to make further changes to this package (apart from maintenance fixes)?

Reply:
No plans for any further changes, other than keeping it going.

As far as I understand it, though, any version lower than 0.1 is considered breaking by the Julia version semantics... So even if we increased this to `0.0.9`, I think all depending packages would have to change their bound. Or to say it another way, I think that for 0.0.x versions, even patch releases are considered breaking by Julia. So if we need to break things in any case, we might as well go to 1.0.0 and then leave this whole pre 1.0.0 sadness behind us, right?

I'll make sure to adjust bounds in the registry for depending packages and will also open PRs against the packages that depend on `Nullables` to adjust their version ranges once we have a new release tagged.

Reply:
Ah, right, because of the `0.0`... so I guess 1.0 is OK.